androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -142,6 +134,7 @@ fun EnhancedVideoPlayer(
                 title = player.mediaMetadata.title?.toString()
                 currentTime = player.contentPosition
                 totalDuration = player.duration
+                loop = player.repeatMode == ExoPlayer.REPEAT_MODE_ALL
```

Comment:
nit: maybe rename it to something like `isLooping` or `isLoopEnabled` to denote that this is a boolean